File: backend/slotify/spring/src/main/java/com/slotify/backend/spring/reserve/dtos/CreateReserveResponse.java
package com.slotify.backend.spring.reserve.dtos;

import lombok.Builder;
import lombok.Getter;
import lombok.ToString;

import java.time.LocalDateTime;
import java.util.UUID;

@Builder
@Getter
@ToString
public class CreateReserveResponse {
    private UUID reserveId;

    private LocalDateTime reserveDateTime;

    private String companyName;

    private String serviceName;

}

File: frontend/app/src/main/java/com/raj/slotify/dtos/reserves/CreateReserveResponse.kt
package com.raj.slotify.dtos.reserves

import com.google.gson.annotations.SerializedName
import java.time.LocalDateTime
import java.util.UUID

data class CreateReserveResponse(

    @SerializedName("reserveId")
    val reserveId: UUID,

    @SerializedName("reserveDateTime")
    val reserveDateTime: LocalDateTime,

    @SerializedName("companyName")
    val companyName: String,

    @SerializedName("serviceName")
    val serviceName: String

)
